web-ui/src/main/resources/catalog/views/default/config.js

```diff
@@ -111,7 +111,8 @@
               'OGC:WMS-1.1.1-http-get-map',
               'OGC:WMS-1.3.0-http-get-map',
               'OGC:WFS',
-              'ESRI:REST'
+              'ESRI:REST',
+              'ESRI REST: Map Server'
```

Comment:
If you don't add the value in ISO19139 list of protocol, that will never be used ?
See https://github.com/geonetwork/core-geonetwork/pull/3914/files#diff-9edb196db9b6857e1f3e578db9328026R2270

Reply:
I remove the entry and test it out, the result is positive. I am going to push the new change shortly. Hopefully I can hear from you about the decision on other subtypes support so I can either add them in the list or leave them for now. Thanks